libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -404,6 +404,17 @@ def write_config(config, pool_file, use_docker, sg_windows, sg_accel_windows,
             }
         }
 
+        if sg_windows:
+            sg_dict = {
+                "sync_gateways:vars": {
+                    "ansible_user": "Administrator",
```

Comment:
We should take this from user. It would be difficult to debug in case username/password is of different combination